lib/Model/index.ts

```diff
@@ -37,7 +37,7 @@ export interface ModelOptions {
 	throughput: "ON_DEMAND" | number | {read: number; write: number};
 	prefix: string;
 	suffix: string;
-	waitForActive: ModelWaitForActiveSettings;
+	waitForActive: false | ModelWaitForActiveSettings;
```

Comment:
Not sure if this should be a boolean instead. I forget what the behavior is for `true`.

Reply:
I thought about that, but I think `true` would have the same behavior as `false` in this case because of this line:

https://github.com/dynamoose/dynamoose/blob/40443acc6f7a07681592f294d3893902c3652e34/lib/Model/index.ts#L318

Because `this.options.waitForActive.enabled` would be `undefined` if I understand correctly

Reply:
@rajivshah3 In that case tho, the documentation needs to be updated to say `false` instead of true. Currently the documentation say it's accepts a boolean, but this says it only accepts `false`.

We probably need to fix that line you linked as well and add some tests.

Reply:
566c42b8ca7807553574e1e802cd811dcf1bff5b